app/controllers/devise_registration_controller.rb

```diff
@@ -21,9 +22,9 @@ class DeviseRegistrationController < Devise::RegistrationsController
   def start
     render :closed and return unless Rails.configuration.enable_registration
 
-    payload = get_payload
+    jwt = find_or_create_jwt
 
-    @criteria_keys = payload.dig(:attributes, :transition_checker_state, "criteria_keys") if payload
+    @criteria_keys = jwt&.jwt_payload&.dig("attributes", "transition_checker_state", "criteria_keys")
```

Comment:
I probably don't have enough context yet - why do we want to use strings instead of symbols? (in a couple of other places too)

Reply:
`payload` used to be `jwt.jwt_payload.deep_symbolize_keys`; but now it's the `jwt`, so the `deep_symbolize_keys` isn't happening any more.

Reply:
I see, thanks for clarifying.